File: kafka-stream-compte/src/main/java/org/enset/kafkastreamcompte/model/Compte.java
package org.enset.kafkastreamcompte.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.Collection;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString

public class Compte {
    private Long id;
    private double solde;
    private Date dateCreation;
    private String type;
    private String etat;
    private Long customerID;
    private Customer customer;
    private Collection<Operation> operations;
}

File: bankApp/src/app/models/Compte.ts
import {Customer} from "./Customer";
import {Operation} from "./Operation";

export class Compte {
  id?: number
  solde?:number
  dateCreation?:string
  type?:string
  etat?:string
  customerID?:number
  customer?:Customer
  operations?:Array<Operation>
}
